timemachine/lib/potentials.py

```diff
@@ -295,24 +277,18 @@ def get_idxs(self):
     def get_rescale_mask(self):
         return self.args[1]
 
-    def get_lambda_plane_idxs(self):
-        return self.args[2]
-
-    def get_lambda_offset_idxs(self):
-        return self.args[3]
-
     def get_beta(self):
-        return self.args[4]
+        return self.args[2]
 
     def get_cutoff(self):
-        return self.args[5]
+        return self.args[3]
 
 
 class NonbondedPairListPrecomputed(CustomOpWrapper):
     """
-    This implements a pairlist with precomputed parameters. It differs from
-    the regular NonbondedPairlist in that it expects params of the form s0*q_ij, s_ij, and s1*e_ij
-    where s are the scaling factor and combining rules have already been applied.
+    This implements a pairlist with precomputed parameters. It differs from the regular NonbondedPairlist in that it
+    expects params of the form s0*q_ij, s_ij, s1*e_ij, and w_offsets, where s are the scaling factors and combining
```

Comment:
nit:
```suggestion
    expects params of the form s0*q_ij, s_ij, s1*e_ij, and w_offsets_ij, where s are the scaling factors and combining
```

(for consistency and to emphasize that this is one offset per pair)